hack/test-e2e.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# should be possible to create and mutate a Service type:LoadBalancer"
+
+# Functions *******************************************************************
+#
+
+function _run_e2e_tests() {
+    echo "Running e2e tests..."
+    # Just run the canary for now... we are limite don loadbalancers.
+    ginkgo -v -progress \
+        -focus "\[Canary\]" \
```

Comment:
Currently, we only run a single test. We can remove this constraint later to enable all the tests.